modules/microshift-tls-config-con.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * microshift_configurig/microshift_tls-config.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-tls-config-con_{context}"]
+= Using TLS with {microshift-short}
+
+Transport layer security (TLS) profiles provide a way for servers to regulate which ciphers a client can use when connecting to the server. Using TLS helps to ensure that {microshift-short} applications use cryptographic libraries that do not allow known insecure protocols, ciphers, or algorithms. You can choose to use either the TLS 1.2 or TLS 1.3 security profiles with {microshift-short}.
+
+{microshift-short} API server cipher suites apply automatically to the following internal control plane components:
+
+* API server
+* Kubelet
+* Kube controller manager
+* Kube scheduler
+* Etcd
```

Comment:
I feel like this should be lowercase, even though it's in a bulleted list, (since it's a Kubernetes term always used in lower case) but I can't find definitive guidance.

Reply:
Hmm, agreed: IBM says, https://www.ibm.com/docs/en/ibm-style?topic=format-lists#capitalization-in-lists
However, they lowercase it everywhere here, even at the start of sentences, https://www.ibm.com/think/topics/etcd  as does etcd's own docs, https://etcd.io/
It is uppercase when used in the API docs, but that seems to be the only true instance where it needs the cap, e.g, `Etcd.parameter` or when it's the object name.

All that to say, I went with LC here.